### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by algodev2**
The 9 files to separate the installation files uploaded for review. Notes:
1. All files need to be reviewed for format, and links updated to .md format
2. Debian and RedHat files are same except for beginning install process.
3. OtherLinux and Mac could be the same, but left Mac separate.
4. The changes should be updated in existing Algorand switch-networks file for making a new service. It shows only instance-template-unit file way.
5. Still need to explain editing template-unit file vs. instance-template-unit file. But didn't do as ConfigureAlgodService as feel it needs much more explaination on systemd file editing. Please decide for me.

### CODE REVIEW

**Comment by algodev2** [comment]
Forgot to remove <pre> and </pre> tags from files. I removed and re-uploading.
Also forgot to include GivePermissionRunAsAlgorandUser.md file

File: docs/run-a-node/setup/InstallANode.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+- [Hardware requirements](#hardware-requirements)
+
+Then install using one of the following:
+
+- [AlgorandDocumentation.InstallANodeOnDebianBasedDistribution](Install a Node on Debian based distributions)
+- [AlgorandDocumentation.InstallANodeOnRedHatBasedDistribution](Install a Node on Red Hat based distributions)
+- [AlgorandDocumentation.InstallANodeOnOtherLinux](Install a Node on Other Linux)
+- [AlgorandDocumentation.InstallANodeOnMac](Install a Node on a Mac)
+
```
Comment (onetechnical) [inline docs/run-a-node/setup/InstallANode.md:21]: These URLs above don't seem right.

File: docs/run-a-node/setup/InstallANode.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+After installation is complete, the following configurations are available for customization:
+
+- [AlgorandDocumentation.GivePermissionsRunAsAlgorandUser](Give user permissions to run as algorand user)
+- [AlgorandDocumentation.InstallANodeConfigureAlgodService](Configure algod as a systemd service)
+- [AlgorandDocumentation.InstallANodeConfigureNodeTelemetry](Configure node to use Telemetry)
+- [AlgorandDocumentation.InstallANodeSyncNodeFastCatchup](Sync node using Fast Catchup)
+- [AlgorandDocumentation.InstallANodeSwitchNetworks](Switching Networks - BetaNet, TestNet, MainNet)
+
```
Comment (onetechnical) [inline docs/run-a-node/setup/InstallANode.md:29]: Same with these - I think they're not linking to the right place/formatted correctly.

File: docs/run-a-node/setup/GivePermissionsRunAsAlgorandUser.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+title: Give Permissions to User to Run as Algorand User
+
+After installing with package manager for Debian or Red Hat distributions and want to customize a different user to be run algorand goal commands requires elevated permissions.
```
Comment (onetechnical) [inline docs/run-a-node/setup/GivePermissionsRunAsAlgorandUser.md:3]:
```suggestion
After installing with package manager for Debian or Red Hat distributions you may want to customize a different user to be able to run algorand goal commands, which requires elevated permissions for that user.

```

File: docs/run-a-node/setup/GivePermissionsRunAsAlgorandUser.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+``` 
+sudo update-alternatives --config editor
+``` 
+
+Check the directory first before making any changes. Confirm which files are there already. Confirm after edit too.
+``` 
+ls -la /etc/sudoers.d/
+``` 
+
+Then vi a new file named algodevuser
```
Comment (onetechnical) [inline docs/run-a-node/setup/GivePermissionsRunAsAlgorandUser.md:30]:
```suggestion
Then edit a new file named algodevuser

```

File: docs/run-a-node/setup/GivePermissionsRunAsAlgorandUser.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+The name `algodevuser` is arbitrary, but to give relevancy to you later. Change as necessary.
```
Comment (onetechnical) [inline docs/run-a-node/setup/GivePermissionsRunAsAlgorandUser.md:32]:
```suggestion
The name `algodevuser` is arbitrary. Change as appropriate to your user.

```

File: docs/run-a-node/setup/InstallANodeConfigureAlgodService.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
+title: Installing algod as a systemd service
+
+After installing using the Updater script method, there are several shell scripts that are bundled into the tarball that are helpful in running `algod`. One of those is the `systemd-setup.sh` script to create a system service.
+
+- username, user run command that will run algorand.service
```
Comment (onetechnical) [inline docs/run-a-node/setup/InstallANodeConfigureAlgodService.md:5]:
```suggestion
- username, user that will run algorand.service

```

**Review by onetechnical — CHANGES_REQUESTED** [review]
I only did a quick pass through here, but can you check all the link anchors? There are a lot of broken links in here.

**Comment by ryanRfox** [comment]
closing for lack of progress

**Pull request closed without merging**